scripts/add-mock-recent-data.ts

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
should this be tested as part of CI to ensure it doesn't break with changes to the indexer?

Reply:
idk how much others are going to use this. but I added some light coverage to the e2e test script in case.

Reply:
I'm hoping we can just get rid of the indexer soon and switch to using Google's BQ dataset, which should cover our analytics needs